.eslintrc.json

```diff
@@ -0,0 +1,36 @@
+{
+    "root": true,
+    "extends": "eslint:recommended",
+    "rules": {
```

Comment:
Going through the [list](http://eslint.org/docs/rules/), I would also add at least these rules:

* [`brace-style`](http://eslint.org/docs/rules/brace-style)
* [`camelcase`](http://eslint.org/docs/rules/camelcase)
* [`curly`](http://eslint.org/docs/rules/curly)
* [`eol-last`](http://eslint.org/docs/rules/eol-last)
* [`no-use-before-define`](http://eslint.org/docs/rules/no-use-before-define)
* [`quotes`](http://eslint.org/docs/rules/quotes)
* [`unicode-bom`](http://eslint.org/docs/rules/unicode-bom)

```json
{
    "brace-style": "error",
    "camelcase": "error",
    "curly": "error",
    "eol-last": "error",
    "no-use-before-define": "error",
    "quotes": ["error", "single"],
    "unicode-bom": "error"
}
```